### PR TITLE
Add Markdown resources #1.

### PR DESCRIPTION
**Pull request by mlh6118**
This PR is to submit a Markdown.md that was posted in Issues by elitadailyuk.

### CODE REVIEW

File: Resources/Markdown.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Markdown was designed to be readable as-is, without looking like it has been marked up with tags or formatting instructions (like HTML). It is a markup language commonly used for writing simply formatted documents without needing to use a GUI to format text, such as using **bold** and *italics*. Markdown allows you to write using an easy-to-read and easy-to-write plain text format.
+
+Documentation on programming projects is often lacking. Markdown is an important language for developers to master because it provides a 
+means to discuss a project and features using a language that all web 
+browsers and integrated development environments (IDEs) can understand, and its use across applications (GitHub, Slack, Notion, etc.).. Far 
+easier than hypertext markup language (HTML) to learn, basic Markdown can be 
+learned in a few minutes. 
+
+## Links / Resources
+### Cheat Sheet
```
Comment (elitadanilyuk) [inline Resources/Markdown.md:12]: Adding Markdown before 'Cheat Sheet', so `Markdown Cheat Sheet` may make it more clear and understandable.

**Comment by BrendonLH** [comment]
handled the merge conflict of previous md file

File: Resources/Markdown.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/)  
+
+### Communicate using Markdown
+***Description***:
+
+-  Making clean and organized information is important in software collaboration. Using markdown allows developers to communicate information more effectively as well as helps guide and instruct on how to use an application. This allows for better pull requests, issue handeling, and readability. The provided resource will guide you through using markdown to update a plain text file.
+
+***Examples***:
+- Creating headings
+- [] Creating check boxes
```
Comment (elitadanilyuk) [inline Resources/Markdown.md:36]: Is there a reason these brackets are here?

**Review by elitadanilyuk — CHANGES_REQUESTED** [review]
I left two small comments for clarity. They aren't blockers per say, just curious about your thoughts.

File: Resources/Markdown.md (hunk 2 above)
Comment (BrendonLH) [reply Resources/Markdown.md:36]: is it not rendering the checkbox? I see vs code was having issue rendering the checkbox, Its not necessary per say, we can remove

File: Resources/Markdown.md (hunk 1 above)
Comment (BrendonLH) [reply Resources/Markdown.md:12]: ahh ok will revise

File: Resources/Markdown.md (hunk 2 above)
Comment (BrendonLH) [reply Resources/Markdown.md:36]: I removed the brackets, its under a new pull request as im not over of the forked repo

File: Resources/Markdown.md (hunk 1 above)
Comment (BrendonLH) [reply Resources/Markdown.md:12]: added Markdown before Cheat under new pull request as I'm not owner of the forked repo

File: Resources/Markdown.md (hunk 2 above)
Comment (elitadanilyuk) [reply Resources/Markdown.md:36]: I'll take a look!  I think in markdown there need's to be a space between the brackets for it to render.  I am happy with it just being a bullet point.

I'll take a look the other PR!

**Comment by elitadanilyuk** [comment]
Closing this PR because it was fixed in PR #16.

**Pull request closed without merging**